db/migrate/20221025191732_create_posts.rb

```diff
@@ -1,8 +1,8 @@
-class CreatePost < ActiveRecord::Migration[7.0]
+class CreatePosts < ActiveRecord::Migration[7.0]
   def change
     create_table :posts do |t|
       t.string :title
-      t.text :text
+      t.string :text
       t.integer :comments_counter
```

Comment:
Please correct this line like this to remove the bug

`t.integer :comments_counter, default: 0`